tests/test_affine_transform.py

```diff
@@ -350,19 +350,19 @@ def test_forward_2d(self):
         expected = torch.nn.functional.grid_sample(x, grid, align_corners=False)
```

Comment:
line 226 also needs to change: `align_corners=True`

Reply:
thanks, this is now fixed on this branch https://github.com/Project-MONAI/MONAI/pull/5860/commits/f9643c37c4efa999c94939b705826a00dccd2e4c